AveryLabels.py

```diff
@@ -25,6 +25,8 @@
     # 22x 32mm x 10mm mini labels
     3044: ( 2, 11, (32, 10), (2,2), (1, 1), (66.5*mm, 120.5*mm)),
 
+    # 189x miniLabel
+    4731: ( 7, 27, (25.4, 10), (2.5,0), (9, 13.5), A4),
```

Comment:
Could you please verify this is working for you without specifying the measurement unit?
```suggestion
    4731: ( 7, 27, (25.4*mm, 10*mm), (2.5*mm, 0), (9*mm, 13.5*mm), A4),
```